packages/npm-mongo/wrapper.js

```diff
@@ -1,6 +1,36 @@
-const useLegacyMongo = !!Package['npm-mongo-legacy'];
+const { MongoClient, MongoCompatibilityError } = Npm.require('mongodb');
 
+function connect(client) {
+  return client.connect()
+    .catch(error => {
+      if (error.cause instanceof MongoCompatibilityError && error.message.includes('maximum wire version')) {
+      console.warn(`[DEPRECATION] Legacy MongoDB version detected, using mongo-legacy package: ${error.message}
+        Warning: MongoDB versions <= 3.6 are deprecated. Some Meteor features may not work properly with this version.
+        It is recommended to use MongoDB >= 4.`);
+      if (!Package['npm-mongo-legacy']) {
+        throw new Error('Please, install npm-mongo-legacy package to use this version of MongoDB running "meteor add npm-mongo-legacy", then move the listed package inside .meteor/packages to the top.');
+      }
+      return false
+    } else {
+      // log the stack
+      console.error(error.stack);
+      throw new Error(`Failed to initialize Meteor's npm-mongo package: ${error}`);
+    }
+  })
+}
+
+if(process.env.MONGO_URL && process.env.MONGO_URL.startsWith('mongodb://')){
```

Comment:
The check process.env.MONGO_URL.startsWith('mongodb://') will skip mongodb+srv://… connection strings entirely. 

Could this work just with `process.env.MONGO_URL.startsWith('mongodb')`?